File: tests/fixtures/go/net/conn.go
// Package net is a second connection layer, exporting a function with the
// same name as db.Open. That ambiguity is the whole point: tier A cannot
// decide which Open a caller in a third file means, and gives up rather than
// guessing. A compiler can.
package net

import "strings"

// Open opens a socket.
func Open(url string) bool {
	return strings.HasPrefix(url, "tcp://")
}

File: tests/fixtures/python/net/conn.py
"""A second connection layer, exporting a function with the same name as
db.conn.open. That ambiguity is the whole point: tier A cannot decide which
open a caller in a third file means, and gives up rather than guessing. A
type checker can.
"""


def open(url: str) -> bool:
    """Open a socket."""
    return url.startswith("tcp://")
